src/textual/scroll_view.py

```diff
@@ -7,11 +7,11 @@
 
 from rich.console import RenderableType
 
+from .containers import ScrollableContainer
 from .geometry import Size
-from .widget import Widget
 
 
-class ScrollView(Widget):
+class ScrollView(ScrollableContainer):
     """
     A base class for a Widget that handles it's own scrolling (i.e. doesn't rely
```

Comment:
```suggestion
    A base class for a Widget that handles its own scrolling (i.e. doesn't rely
```

Reply:
I'd normally discourage fixing unrelated things in a PR, but seeing as it's one character I'll let it slide. ;-)